bazel_tools/haskell-package-env.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/haskell/private/actions/compile.bzl b/haskell/private/actions/compile.bzl
```

Comment:
We should try to upstream that patch. Just haven’t had the time so far sadly :disappointed:

Reply:
I tried to just apply the patch on rules_haskell. Sadly it breaks plugins, haddock, and doctests.